docs/authentication/saml/google.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: Integrate Google SAML with Clerk
+description: Learn how to integrate .
+---
+
+# Integrate Google SAML with Clerk
+
+<TutorialHero
+  beforeYouStart={[
+    {
+      title: "Set up a Clerk application",
+      link: "/docs/quickstarts/setup-clerk",
+    },
+    {
+      title: "Set up a local project",
+      link: "/docs/quickstarts/overview",
+    },
+  ]}
+>
+
+- Use Google SAML to authenticate users in your Clerk application.
+
+</TutorialHero>
+
+Integrating Google SAML with Clerk process involves setting up a SAML application in Google Cloud and configuring it to work with Clerk. This tutorial will guide you through the process of setting up Google SAML and integrating it with Clerk.
+
+## Tutorial
+
+<Callout type="info">
+  You must have a local project with Clerk set up already to follow this tutorial. See [the quickstart docs](/docs/quickstarts/overview) to get started.
+</Callout>
+
+
+<Steps>
+
+### Setup Clerk SAML Connection
+1. Create a new Enterprise Connection in the Clerk Dashboard. 
+2. Fill out the form with the details for your Google Application.
+
+<Images 
+  width={3024}
+  height={1652}
+  src="/docs/images/authentication/saml/.avif"
+  alt="Create a new Enterprise Connection in the Clerk Dashboard"
+/>
+
+### Find the Google Enterprise Applications 
+
+<Images 
+  width={3024}
+  height={1652}
+  src="/docs/images/authentication/saml/.avif"
+  alt=""
+/>
+
+###  Create a new Google Enterprise Application
+
+<Images 
+  width={3024}
+  height={1652}
+  src="/docs/images/authentication/saml/.avif"
+  alt=""
+/>
+
+
+#### Follow the Google SAML Setup Wizard
+
+<Images 
+width={3024} 
+height={1652} 
+src="/docs/images/authentication/saml/.avif" 
+/>
+
+
+<Images 
+width={3024} 
+height={1652} 
+src="/docs/images/authentication/saml/.avif" 
+/>
+
+<Images 
+width={3024} 
+height={1652} 
+src="/docs/images/authentication/saml/.avif" 
+/>
+
+SAML Configuration in Google 
+
+<Images
+  width={3024}
+  height={1652}
+  src="/docs/images/authentication/saml/.avif"
+  alt=""
+/>
+
+#### Clerk Reference for SAML Settings
+
+<Images
+  width={3024}
+  height={1652}
+  src="/docs/images/authentication/saml/.avif"
+  alt=""
+/>
+
+### Attribute Mapping in Google to Clerk 
+
+You will want to add the following fields to the attribute mapping in Google to Clerk.
+<Images 
+width={3024} 
+height={1652} 
+src="/docs/images/authentication/saml/.avif"
+alt="Attribute Mapping in Google to Clerk" 
+/>
+
+
+
+<Images 
+  width={3024}
+  height={1652}
+  src="/docs/images/authentication/saml/.avif"
+  alt=""
+/>
+
+### Google PEM Certificate Download & Upload to Clerk
```

Comment:
We should guide users to download the connection metadata file and use that to configure it in the Clerk Dashboard instead of the manual process

Reply:
Mmmm I see, yeah during your Demo I noticed the one from Sokratis was a bit outdated